src/engine/Settings.cs

```diff
@@ -63,75 +111,123 @@ private Settings()
     /// </summary>
     public bool PlayMicrobeIntroVideo { get; set; } = true;
 
-    /// <summary>
-    ///   If true an auto-evo run is started during gameplay,
-    ///   taking up one of the background threads.
-    /// </summary>
-    public bool RunAutoEvoDuringGamePlay { get; set; } = true;
-
     /// <summary>
     ///   If false auto saving will be disabled
     /// </summary>
     public bool AutoSaveEnabled { get; set; } = true;
 
-    /// <summary>
-    ///   Number of quick saves to keep
-    /// </summary>
-    public int MaxQuickSaves { get; set; } = 5;
-
     /// <summary>
     ///   Number of auto saves to keep
     /// </summary>
     public int MaxAutoSaves { get; set; } = 5;
 
     /// <summary>
-    ///   This can be freely adjusted to adjust the performance The
-    ///   higher this value is the smaller the size of the simulated
-    ///   cloud is and the performance is better.
-    /// </summary>
-    public int CloudResolution { get; set; } = 2;
-
-    /// <summary>
-    ///   If this is over 0 then this limits how often compound clouds
-    ///   are updated. The default value of 0.020 at 60 FPS makes
-    ///   every other frame not update the clouds.
+    ///   Number of quick saves to keep
     /// </summary>
-    /// <remarks>
-    ///   <para>
-    ///     This should be made user configurable for different
-    ///     computers. The choises should probably be:
-    ///     0.0f, 0.020f, 0.040f, 0.1f, 0.25f
-    ///   </para>
-    /// </remarks>
-    public float CloudUpdateInterval { get; set; } = 0.040f;
+    public int MaxQuickSaves { get; set; } = 5;
 
     /// <summary>
-    ///   Sets amount of MSAA to apply to the viewport
+    ///   When true cheats are enabled
     /// </summary>
-    public Viewport.MSAA MSAAResolution { get; set; } = Viewport.MSAA.Disabled;
+    public bool CheatsEnabled { get; set; } = false;
 
     /// <summary>
-    ///   Choose what filter to apply to the screen
-    ///   0 = None, 1 = Red/Green, 2 = Blue/Yellow
+    ///   If true an auto-evo run is started during gameplay,
+    ///   taking up one of the background threads.
     /// </summary>
-    public int ColourblindSetting { get; set; } = 0;
+    public bool RunAutoEvoDuringGamePlay { get; set; } = true;
```

Comment:
`RunAutoEvoDuringGamePlay` is a performance option and should be grouped with them.

Reply:
It's actually not even shown in the options menu right now, it was just there and always enabled in the settings. So I didn't actually think of it as being a performance option since I wasn't really sure what it did. That's why I had it at the bottom.

Reply:
I was too lazy to add it to the options menu.
For people with only a dual core CPU it might help once auto-evo runs for more like 30 seconds (instead of 0.1), to only start auto-evo run once they get to the editor so that the gameplay isn't a laggy mess.

Reply:
I could pop an option for it on there if you want. If it's not really functioning right now I'm not sure it makes much difference though.

Reply:
I don't think an option is needed before it really works, it would need to be set to disabled state anyway if added.